internal/endorser/endorser.go

```diff
@@ -90,9 +99,15 @@ func loadAndVerifyProvenances(referenceValues *common.ReferenceValues, provenanc
 		return nil, fmt.Errorf("verification of provenances failed: %v", result.Justifications)
 	}
 
+	refBinaryDigest := provenanceIRs[0].GetBinarySHA256Digest()
+	refBinaryName, err := provenanceIRs[0].GetBinaryName()
```

Comment:
I think `GetBinaryName` can be changed to only return `string`. Then these variable declarations will not be needed.